src/pf-notification-drawer/pf-notification-drawer-accordion.component.js

```diff
@@ -0,0 +1,51 @@
+import {pfUtil} from 'pf-utils.js';
+
+/*
+ * <b>&lt;pf-notification-drawer-accordion&gt;</b> element for Patternfly Web Components
+ *
+ * @example {@lang xml}
+ * <pf-notification-drawer-accordion>
+ *   <pf-notification-drawer-accordion-panel></pf-notification-drawer-accordion-panel>
+ *   <pf-notification-drawer-accordion-panel></pf-notification-drawer-accordion-panel>
+ *   . . .
+ * </pf-notification-drawer-accordion>
+ */
+
+export class PfNotificationDrawerAccordion extends HTMLElement {
+
+  /*
+   * An instance of the element is created or upgraded
+   */
+  constructor () {
+    super();
+  }
+
+  /*
+   * Called every time the element is inserted into the DOM
+   */
+  connectedCallback () {
+    let that = this;
+    pfUtil.addClass(this, 'panel-group');
+
+    let observer = new MutationObserver(function(mutations) {
```

Comment:
@priley86 I know you had reservations related to using MutationObserver and performance. Are we ok using MutationObserver?